.github/renovate.json

```diff
@@ -87,6 +87,7 @@
     "apis/Google.Cloud.LifeSciences.V2Beta/Google.Cloud.LifeSciences.V2Beta/**",
     "apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/**",
     "apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/**",
+    "apis/Google.Cloud.Location/Google.Cloud.Location/**",
```

Comment:
Is still out of order here, which is curious since this is generated? Just flagging in case there's something else missing.